playbook/app/pb_kits/playbook/pb_rich_text_editor/_rich_text_editor.tsx

```diff
@@ -4,6 +4,7 @@ import inlineFocus from './inlineFocus'
 import useFocus from './useFocus'
 import { globalProps, GlobalProps } from '../utilities/globalProps'
 import { buildAriaProps, buildDataProps, noop } from '../utilities/props'
+import cn from 'classnames'
```

Comment:
@augustomallmann Looks like we are already importing this. So no need to bring it in again.